pages/Frequency/Publishing.md

```diff
@@ -1,24 +1,56 @@
+| 4 | [Reaction](../DSNP/Types/Reaction.md) | YES | TBD | TBD | [`Parquet`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.ModelType.html#variant.Parquet) | [`IPFS`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.IPFS) |
+| 5 | [Profile](../DSNP/Types/Profile.md) | YES | TBD | TBD | [`Parquet`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.ModelType.html#variant.Parquet) | [`IPFS`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.IPFS) |
+| 6 | [Update](../DSNP/Types/Update.md) | YES | TBD | TBD | [`Parquet`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.ModelType.html#variant.Parquet) | [`IPFS`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.IPFS) |
 
-## Batch File Retrieval
+Source code for each schema is also located in the [LibertyDSNP/schemas](https://github.com/LibertyDSNP/schemas) repository.
 
-- Batch File URLs MUST NOT refer to localhost or any reserved IP addresses as defined in [RFC6890](https://datatracker.ietf.org/doc/html/rfc6890).
-- Batch File URLs MUST use one of the supported URL Schemes.
+## DSNP Non-Batched Announcements
 
-### Supported URL Schemes
+Frequency uses an on-chain data structure for storage of `GraphChange` Announcements.
+Each individual announcement is submitted using the [`add_onchain_message`](https://libertydsnp.github.io/frequency/pallet_messages/pallet/enum.Call.html#variant.add_onchain_message) extrinsic.
 
-| Scheme | Description | Reference | DSNP Version Added |
-| ------ |------------ | --------- | ------------------ |
-| HTTPS | Hypertext Transfer Protocol Secure | [RFC2818](https://datatracker.ietf.org/doc/html/rfc2818) | 1.0 |
+## DSNP Batched Announcements
+
+Frequency uses [DSNP Batch Publications](../DSNP/BatchPublications.md) for Announcements that are batched.
+The parquet file is stored on IPFS, but it is discovered through the Frequency Message.
+
+Frequency Messages maintain the data of when the data was published and what Providers MSA published it.
+It is the publisher's responsibility to maintain the IPFS pin so that the batch file is continuously available.
+
+DSNP Batch Publications [MUST be validated](./Validation.md) upon fetching to ensure data and permission integrity.
 
 ## Ordering
 
-...
+Frequency Messages are well ordered within a Schema
+
+1. Frequency: Block number ascending
+2. Frequency: Transaction index ascending
+3. DSNP Standard: Order Announcements in a Batch Publication File by row appearance order
+
+### Ordering Across Schemas
+
+Frequency does not provide complete order information for Messages across Schemas.
```

Comment:
We do provide order across different schemas in the same block. `Index` stored on each Message is a total index in that block across all schemas.